packages/expo-error-recovery/ios/EXErrorRecovery/EXErrorRecoveryModule.h

```diff
@@ -0,0 +1,14 @@
+//  Copyright © 2018 650 Industries. All rights reserved.
+
+#import <UMCore/UMExportedModule.h>
+#import <UMCore/UMModuleRegistryConsumer.h>
+
+static NSString * const USER_DEFAULTS_KEY = @"expo.errorRecovery";
```

Comment:
1. It will work as long as we didn't have a file which imports this header twice - from two different versioned abis.

2. I think it's a good idea to make the value of this variable depends on the current SDK version.